server/grpc_service.go

```diff
@@ -788,19 +788,26 @@ func (s *GrpcServer) ReportBuckets(stream pdpb.PD_ReportBucketsServer) error {
 		if err := s.validateRequest(request.GetHeader()); err != nil {
 			return err
 		}
-		start := time.Now()
-		bucketReportCounter.WithLabelValues("report", "recv").Inc()
 		buckets := request.GetBuckets()
 		if buckets == nil || len(buckets.Keys) == 0 {
 			continue
 		}
-		err = rc.HandleBucketHeartbeat(request.Buckets)
+		store := rc.GetLeaderStoreByRegionID(buckets.GetRegionId())
+		if store == nil {
+			return errors.Errorf("region's store %d not found", buckets.GetRegionId())
```

Comment:
The error message is not clear

Reply:
how about `the store of the bucket %v is not found`

Reply:
`%v` actually is region ID

Reply:
the bucket has no constant ID and it will report all buckets once that belongs one same region.

Reply:
How about `the store of the bucket in region %v is not found`

Reply:
fix